Book-A-Desk.Api/SmtpClientManager.fs

```diff
@@ -0,0 +1,20 @@
+namespace Book_A_Desk.Api
+
+open System.Net.Mail
+
+type SmtpClientManager =
```

Comment:
What's the purpose of SmtpClientManager? I think in this case it would be simpler to simply return the SmtpClient

Reply:
SmtpClientManager will manage turn the certificate validation off for the SmtpClient Instance. It will look cleaner in the BookingNotifier file.